README.md

```diff
@@ -1,135 +1,268 @@
+
+- Specification writers: the FAW makes it easy to compare various
+  parsers that implement the spec as well as examine the kind of
+  documents that are available in the wild and how they interpret the
+  specification.
+- Parser Developers: in addition to the parser comparisons mentioned
+  above, the FAW also makes it easy to make changes to a parser and test
+  parser behavior against large corpus.
+- Vulnerability Researchers: the FAW allows easy incorporation of new
+  ananlysis to dig more in to the programs and the specification under
+  test.
+
+# Screenshot
 
 Main overview of decisions
 ![Main overview](./docs/readme/screenshot-main.png)
 
-Decision DSL excerpt
-![DSL excerpt](./docs/readme/screenshot-dsl.png)
+# Architecture of the FAW
+
+Exploring data formats via input corpora at scale requires an
+infrastructure to run parsers and associated tools against input files
+in the corpus and capture their behavior. The FAW is divided in to a
+backend that handles the details of running parsers against the input
+corpus, and a frontend that enables users to investigate the behavior of
+parsers and tools against files in the corpus, at different scales and
+granularities.
+
+Parsers and parser-like tools are provided as binary executables and can
+be written in any language, and using any technology. Further FAW
+supports processing the file corpus either on a single machine or in
+parallel on a cluster of machines. The latter, termed a "teaming" setup,
+can be performed on a private network without requiring internet access.
+
+The FAW allows the user to specify analysis sets to control the scale
+and speed of analyses. It also provides multiple tools are provided to
+explore the interactions between inputs and parsing programs in the
+analysis set.
+
+TODO: Note on features as booleans.
+
+![FAW Architecture](./docs/readme/faw-arch.png)
+
+# Quick Start
+
+Running the FAW requires an installation of Python (\>= 3.8). In order
+to start an instance:
+
+- Clone the repository
+- Install the prerequisites with `pip3 install -r requirements.txt`
+- Launch the startup script `workbench.py` with two paths: the path to a
+  *distribution folder* and one to the file corpus being studied.
+
+A FAW distribution is a specific configuration of the FAW and is usually
+associated with a specific file format or corpus. The FAW repository
+includes a small set of distributions and corresponding file corpora,
+for example PDF. The PDF distribution can be launched from the root
+folder with the following command: `workbench.py pdf test_files/pdf`.
+
+The workbench script builds a docker image for the distribution on the
+fly and launches an instance of the FAW. The FAW user interface is
+accessible via a webbrowser at `http://localhost:8123` (by default).
+
+By default, FAW is launched in [developer mode](#developer-friendly-features).
+
+To create your own distribution, take a look at
```

Comment:
Maybe add "for a new file format"

Reply:
The line has been moved and updated. Please take a look at [this section](https://github.com/GaloisInc/FAW/tree/faw-readme#distribution) and see if it looks ok?